libshvchainpack/src/chainpack/cpon.h

```diff
@@ -1,30 +1,30 @@
 #pragma once
 
-#include <string>
+#include <string_view>
 
 namespace shv {
 namespace chainpack {
 
 class Cpon
 {
 public:
-	static const std::string STR_NULL;
-	static const std::string STR_TRUE;
-	static const std::string STR_FALSE;
-	static const std::string STR_IMAP_BEGIN;
-	//static const std::string STR_ARRAY_BEGIN;
-	static const std::string STR_ESC_BLOB_BEGIN;
-	static const std::string STR_HEX_BLOB_BEGIN;
-	static const std::string STR_DATETIME_BEGIN;
+	static constexpr auto STR_NULL = "null";
```

Comment:
my understanding is, that this introduces "null", "true", "false", ... strings into every compilation unit, where `cpon.h` is included.

Reply:
maybe it has changed in c++17?